styles.css

```diff
@@ -122,6 +122,20 @@ main {
   border: 3px solid #FD8078;
 }
 
+.log-activity-button {
```

Comment:
You did great work with this log activity button, this was a challenging one! Nice work!